extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss

```diff
@@ -17,10 +17,6 @@ $color-borders: #CECECE;
 	}
 }
 
-.CorporateFooter {
-	display: none;
-}
-
 .WikiaFooter {
```

Comment:
This container is now empty so maybe it doesn't make sense to keep this class here?